# called on the VOS Client by the VOS Server: the VOS Server is tracking the robot, and makes this call for each localisation of the robot, to the VOS Client running on the robot

nav_msgs/Odometry odometry
---
string  acknowledgement

